1/test/App.spec.js

```diff
@@ -0,0 +1,14 @@
+import {mount, shallowMount} from '@vue/test-utils'
+import app from '../src/App.vue'
+
+describe('App.vue', () => {
+    const wrapper = mount(app);
+    it('renders the todo app', () => {
+        expect(wrapper.contains('div')).toBe(true);
+    })
+
+    it('contains data', () =>{
```

Comment:
It's not how the test case is implemented. The test does not check if there is data, just if there is a header with `TodoList`.